components/merkledb/README.md

```diff
@@ -0,0 +1,84 @@
+# exonum-merkledb
+
+[![Travis Build Status](https://img.shields.io/travis/exonum/exonum/master.svg?label=Linux%20Build)](https://travis-ci.com/exonum/exonum)
+[![Docs.rs](https://docs.rs/exonum-merkledb/badge.svg)](https://docs.rs/exonum-merkledb)
+![rust 1.33.0+ required](https://img.shields.io/badge/rust-1.33.0+-blue.svg?label=Required%20Rust)
+[![License: Apache-2.0](https://img.shields.io/github/license/exonum/exonum.svg)](https://github.com/exonum/exonum/components/merkledb/blob/master/LICENSE)
+
+Exonum-merkledb is a persistent storage implementation based on RocksDB and
+provides APIs to work with merkelized data structures.
```

Comment:
Since "Merkle tree" is named after Ralph Merkle, I think "merklized data structures" will be more correct.
However, I google prefers "merkelized" and I'm not sure about that.

Reply:
I use "merkelized" since we already have "merkelized list" in our documentation. https://exonum.com/doc/version/latest/advanced/merkelized-list/. But I'm also not sure which variant is more correct.

Reply:
As far as I understand, this error comes from [this](https://github.com/bitcoin/bips/pull/368/files) BIP.
It's called "Merkelized Abstract Syntax Tree", but if you'll look inside you'll notice that author uses "Merkel root". So I think that it's an origin of this mistake.

Anyway, I guess it's a topic for further investigation.